VirtoCommerce.DynamicExpressionsModule.Data/Promotion/Rewards/RewardItemForEveryNumOtherItemInGetOfRel.cs

```diff
@@ -0,0 +1,31 @@
+using VirtoCommerce.Domain.Marketing.Model;
+
+namespace VirtoCommerce.DynamicExpressionsModule.Data.Promotion
+{
+    //For [] items of entry [] in every [] items of entry [] get [] % off no more than [] not to exceed $ []
+    public class RewardItemForEveryNumOtherItemInGetOfRel : RewardItemForEveryNumInGetOfRel
+    {
+        public ProductContainer ConditionalProduct { get; set; } = new ProductContainer();
+
+        #region IRewardExpression Members
+
+        public override PromotionReward[] GetRewards()
+        {
+            return new PromotionReward[] {
```

Comment:
Can we reuse reward object already created in base class  and just set ConditionalProduct to it?

Reply:
Yes, we can. Did not make it that way initially as in that case we would have a tight dependency on base class implementation. Thought about this again - probably we want the same values to be filled with the same fields in the same order in our case, so yes, better to use base method.